function-images/tests/chained-function-serving/docker-compose.yml

```diff
@@ -40,8 +47,10 @@ services:
             - -addr=consumer
             - -ps=3031
             - -pc=3030
+            - -zipkin=http://zipkin:9411/api/v2/spans
```

Comment:
make sure that you open this port (or any other ports that are used in the code) to all containers. You can start by debugging this.

Reply:
When trying to expose port 9411 to all containers I receive a "port is already allocated" error